.github/workflows/container-deploy.yml

```diff
@@ -60,10 +60,19 @@ jobs:
           cd openfoodfacts-server/docker/
 
           # Pull latest container images
-          sudo docker-compose pull
+          sudo docker pull node:lts
+          sudo docker pull ghcr.io/openfoodfacts/openfoodfacts-server/backend:latest
+          sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.dev.yml -f ./docker-compose.deploy-test.yml pull
+
+          # Rebuild npm
+          sudo bash ./build_npm.sh
```

Comment:
see discussions on using pre-built containers instead of re-building the frontend assets